iox_query_influxql/src/plan/planner.rs

```diff
@@ -1112,7 +1114,7 @@ impl<'a> InfluxQLToLogicalPlan<'a> {
 
         match ctx.scope {
             ExprScope::Where => {
-                if call.name.eq_ignore_ascii_case("now") {
+                if is_now_function(&call.name.to_lowercase()) {
```

Comment:
```suggestion
                if is_now_function(&call.name) {
```

I should have updated this, as there is no need to call `to_lowercase`, as parsing the `call` expression always converts the name to lowercase:

https://github.com/influxdata/influxdb_iox/blob/2dddda7c3366de3c5d481acee35fec08b4472a97/influxdb_influxql_parser/src/expression/arithmetic.rs#L461

I do have some tests to validate the name is in lower case in the display form:
https://github.com/influxdata/influxdb_iox/blob/2dddda7c3366de3c5d481acee35fec08b4472a97/influxdb_influxql_parser/src/expression/arithmetic.rs#L907-L920